ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -100,6 +102,169 @@ enum InstitutionMode {
+  const {institution, onChange} = props;
+  return <InputSwitch
+      data-test-id='controlled-enabled-switch'
+      onChange={(v) => onChange(v.value)}
+      checked={getControlledTierConfig(institution).membershipRequirement !== InstitutionMembershipRequirement.NOACCESS}
+      disabled={false} // TODO
+  />;
+};
+
+const RequirementDropdown = (props: {tierConfig: InstitutionTierConfig, onChange: (InstitutionMembershipRequirement) => void}) => {
+  const {tierConfig, onChange} = props;
+  return <Dropdown style={{width: '16rem'}}
+                   data-test-id={`${tierConfig.accessTierShortName}-agreement-dropdown`}
+                   placeholder='Select type'
+                   options={MembershipRequirements}
+                   value={tierConfig.membershipRequirement}
+                   onChange={(v) => onChange(v.value)}/>;
+};
+
+const AddressTextArea = (props: {accessTierShortName: string, emailAddresses: string[], onBlur: Function, onChange: (string) => void}) => {
+  const {accessTierShortName, emailAddresses, onBlur, onChange} = props;
+  return <TextArea
+      value={emailAddresses && emailAddresses.join(',\n')}
+      data-test-id={`${accessTierShortName}-email-address-input`}
+      onBlur={onBlur}
+      onChange={onChange}/>;
+};
+
+const DomainTextArea = (props: {accessTierShortName: string, emailDomains: string[], onBlur: Function, onChange: (string) => void}) => {
+  const {accessTierShortName, emailDomains, onBlur, onChange} = props;
+  return <TextArea value={emailDomains && emailDomains.join(',\n')}
+                   data-test-id={`${accessTierShortName}-email-domain-input`}
+                   onBlur={onBlur}
+                   onChange={onChange}/>;
+};
+
+interface TierConfigProps {
+  institution: Institution;
+  accessTierShortName: string;
+  setEnableControlledTier?: (boolean) => void;
+  setEraRequired: (boolean) => void;
+  setTierRequirement: (InstitutionMembershipRequirement) => void;
+  validateTierAddresses: Function;
+  setTierAddresses: (string) => void;
+  validateTierDomains: Function;
+  setTierDomains: (string) => void;
+  invalidAddress: boolean;
+  invalidAddressMsg: string;
+  invalidDomain: boolean;
+  invalidDomainMsg: string;
+}
+const TierConfig = (props: TierConfigProps) => {
+  const {institution, accessTierShortName, setEnableControlledTier, setEraRequired, setTierRequirement,
+    validateTierAddresses, setTierAddresses, validateTierDomains, setTierDomains,
+    invalidAddress, invalidAddressMsg, invalidDomain, invalidDomainMsg} = props;
+
+  const tierConfig = getTierConfig(institution, accessTierShortName);
+  const {emailAddresses, emailDomains} = tierConfig;
+
+  return <FlexRow style={styles.tierConfigContainer}>
+    <FlexColumn>
+      {accessTierShortName === AccessTierShortNames.Registered
+          ? <RegisteredTierBadge style={{marginTop: '0.6rem', marginLeft: '0.6rem'}}/>
+          : <ControlledTierBadge style={{marginTop: '0.6rem', marginLeft: '0.6rem'}}/>}
```

Comment:
nit: I feel this is the one spot that slightly breaks separation of concerns; maybe make the badge content a prop or else change this to be a switchCase? Issue being that there's an implicit assumption that there's only two tiers, i.e. if ! registered, we assume controlled.